Hw2/Task1/PointMass.cs

```diff
@@ -0,0 +1,44 @@
+public class PointMass
+{
+    private int[] _coordinates = new int[3];
+    private double _mass;
+    
+    public int X
+    {
+        get { return _coordinates[0]; }
+        set { _coordinates[0] = value; }
+    }
+
+    public int Y
+    {
+        get { return _coordinates[1]; }
+        set { _coordinates[1] = value; }
+    }
+
+    public int Z
+    {
+        get { return _coordinates[2]; }
+        set { _coordinates[2] = value; }
+    }
+
+    public double Mass 
+    {
+        get { return _mass; }
+        set { _mass = value >= 0 ? value : 0; }
+    }
+
+    public bool IsZero() 
+    {
+        return _coordinates.All(a => a == 0);
+    }
+
+    public double DistanceFrom(PointMass pointMass) 
+    {
```

Comment:
pointMass can be negative

Reply:
No it cannot (mass). There is setter that takes care of that.